### PR TITLE
Fix/app rotation

### PR DESCRIPTION
**Pull request by johnxnguyen**
## Problem
The app was not able to rotate when viewing fullscreen images.

## Cause
The app was in fact not receiving any device rotation information (including `UIDeviceOrientationNotification`'s) from the system. The problem seemed to result in the recent changes to the root view controller instantiation with the introduction of the new `AppRootViewController` class. All window and root view controller objects are managed by this class, and this class was first created in the `AppDelegate`'s initializer. After much trial and error, we discovered that although the view controller hierarchy appeared to be properly constructed, the windows were not receiving any information regarding the device's orientation, and concluded that the windows were being created too early.

## Solution
Simply delaying the instantiation of `RootViewController` to the `application:WillFinishLaunchingWithOptions:` method solved the problem. Furthermore, the rotation handling of the `NotificationWindowRootViewController` needed to be updated to match exactly the `RootViewController`.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Overlay/NotificationWindowRootViewController.m [hunk 1]
```diff
@@ -120,16 +120,16 @@ - (void)transitionToLoggedInSession
     [self.voiceChannelController.view autoPinEdgesToSuperviewEdgesWithInsets:UIEdgeInsetsZero];
 }
 
-#pragma mark - Rotation handling (should match up with roort)
+#pragma mark - Rotation handling (should match up with root)
 
 - (BOOL)shouldAutorotate
 {
-    return UIApplication.sharedApplication.wr_topmostViewController.shouldAutorotate;
+    return YES;
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Overlay/NotificationWindowRootViewController.m]: I think with this change the calling UI would start to rotate, if you receive the call while looking on the picture. The problem is the calling UI is not looking good on iPhone landscape. Please check this case.

Comment (johnxnguyen) [reply Wire-iOS/Sources/UserInterface/Overlay/NotificationWindowRootViewController.m]: ah ok, will do 👍

**Comment by johnxnguyen** [comment]
@mikeger I've just notice that sometimes opening the app classy isn't finished loading before the UI is visible

**Comment by mikeger** [comment]
@johnxnguyen classy issue is annoying, but let's fix the top issues first

**Pull request merged**